appservice/patternfly.css

```diff
@@ -0,0 +1,7846 @@
+.pf-t-light {
```

Comment:
This slipped into git. Please remove, add to .gitignore, and add it to the `clean` rule.

Or was this deliberate? I guess it'd be convenient to not have to run `npm` all the time, but it could just happen as part of the container build.

(See also my comment about pinning down the version).

Reply:
I added it deliberately, we could I guess pin npm and generate it all the when required. I don't have a strong opinion on this issue yet.